#舵机返回角度
uint16 servo_id_1	#舵机ID
uint16 angle_1	#角度位置0~1000
uint16 servo_id_2	#舵机ID
uint16 angle_2	#角度位置0~1000
